posthog/temporal/common/shutdown.py

```diff
@@ -66,16 +69,37 @@ def __str__(self) -> str:
         else:
             return f"<ShutdownMonitor: Worker running>"
 
+    @property
+    def logger(self):
```

Comment:
Is it safe having this as the same name as `logger = structlog.get_logger()` above? 👀

Reply:
Given that this is namespaced to the class instance there are no naming conflicts.